apps/packages/ui/src/assets/locale/en/settings.json

```
@@ -2052,6 +2052,15 @@
       "label": "API Key",
       "placeholder": "Enter your API key (or use the default demo key THIS-IS-A-SECURE-KEY-123-REPLACE-ME for local single-user setups)"
     },
+    "rememberApiKey": {
```

Biome (2.5.1)

[error] The key rememberApiKey was already declared.

(lint/suspicious/noDuplicateObjectKeys)